scripts/bump.js

```diff
@@ -185,6 +210,7 @@ const bumpExtensionPackage = (packageName, versionToBump) => {
 const bumpPackages = (packageName, versionToBump) => {
   if (packageName === 'cli, core, schema') {
     bumpMainPackages(versionToBump);
+    bumpMainPackagesForExampleApps(versionToBump);
```

Comment:
I removed bumping example apps in bf48591e8a to unblock my release of 9.1.0. This only adds it back.